templates/base.html.twig

```diff
@@ -184,6 +184,22 @@
                     {{ algolia_api_key ? ('\''~algolia_api_key|e('js')~'\'')|raw : 'null' }},
                     '{{ algolia_blacklist|base64_encode }}'
                 );
+
+                findAll(document, '.js--tabs').forEach(function (tab) {
```

Comment:
do we have anywhere on the site something to switch tabs ? (I only saw tabs, that are news pages, not on the same page). If not, should this code be added here or somewhere else ?

Reply:
I think we can remove this block